lib/beaker-hostgenerator/generator.rb

```diff
@@ -142,6 +142,10 @@ def unstringify_number_or_boolean(value)
         result = true
       elsif value == 'false'
         result = false
+      elsif value.kind_of?(Array)
+        value.each_with_index do |v, i|
+          result[i] = unstringify_number_or_boolean(v)
```

Comment:
Nice!

We should update this method name though since it now supports more than just numbers and booleans.
Rather than spelling out the supported datatypes in the method name I think we can just call this `unstringify_value` which would go nicely with the function that calls it (`unstringify_values!`).

Also we should update the docstring on it.